packs/mmonit/requirements.txt

```diff
@@ -0,0 +1 @@
+requests
```

Comment:
To be on the safe side we should pin in to a specific set of versions (e.g. `>=2.6.0,<3.0` should work).